backend/src/nodes/image_util_nodes.py

```
@@ -12,110 +12,112 @@
+            ),
+            BlendModeDropdown(),
         ]
         self.outputs = [ImageOutput()]
         self.category = IMAGE_UTILITY
-        self.name = "Overlay Images"
+        self.name = "Blend Images"
         self.icon = "BsLayersHalf"
         self.sub = "Compositing"
 
     def run(
         self,
         base: np.ndarray,
         opbase: float,
-        ov1: np.ndarray,
-        op1: float,
-        ov2: Union[np.ndarray, None],
-        op2: float,
+        ov: np.ndarray,
+        op: float,
+        blend_mode: int,
     ) -> np.ndarray:
-        """Overlay transparent images on base image"""
+        """Blend images together"""
 
         # Convert to 0.0-1.0 range
         opbase /= 100
-        op1 /= 100
-        op2 /= 100
+        op /= 100
 
         imgs = []
-        max_h, max_w = 0, 0
-        for img in base, ov1, ov2:
-            if img is not None:
-                h, w, c = get_h_w_c(img)
-                max_h = max(h, max_h)
-                max_w = max(w, max_w)
+        max_h = max_w = 0
+        for img in (base, ov):
+            h, w, c = get_h_w_c(img)
+            max_h = max(h, max_h)
+            max_w = max(w, max_w)
 
-                # All inputs must be BGRA for alpha compositing to work
-                if c == 1:
-                    img = cv2.cvtColor(img, cv2.COLOR_GRAY2BGRA)
-                elif c == 3:
-                    img = np.dstack((img, np.ones((h, w), np.float32)))
-                elif c != 4:  # Explode if there are not 1, 3, or 4 channels
-                    logger.error(f"Number of channels ({c}) unexpected")
+            # All inputs must be BGRA for alpha compositing to work
+            if c == 1:
+                img = cv2.cvtColor(img, cv2.COLOR_GRAY2BGRA)
+            elif c == 3:
+                img = cv2.cvtColor(img, cv2.COLOR_BGR2BGRA)
+            elif c != 4:  # Explode if there are not 1, 3, or 4 channels
+                logger.error(f"Number of channels ({c}) unexpected")
```

Could you throw an error here? Or make an assertion before the if/elifs?